### PR TITLE
Add new GWAS term request SOP

### LINKED ISSUE #154: Write an SOP on how to add new GWAS requested trait terms to OBA and EFO
- [ ] Write the SOP
- [ ] Get feedback
- [ ] Improve it

### PR DESCRIPTION
**Pull request by rays22**
on how to add new GWAS requested trait terms to OBA and EFO.

If applied, this commit will fix #154.

### CODE REVIEW

File: docs/SOP-add_OBA_terms_for_GWAS.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+* Merge as many open PRs as possible
+* Start with a fresh copy of the `master` branch
+
+    git pull
+
+* Checkout a new branch
+
+    git checkout -b release-202X-XX-XX
+
+* In a terminal window, start the [release pipeline:](http://pato-ontology.github.io/pato/odk-workflows/ReleaseWorkflow/)
```
Comment (matentzn) [inline docs/SOP-add_OBA_terms_for_GWAS.md:39]: Wouldnt it be better to create our own?

File: docs/SOP-add_OBA_terms_for_GWAS.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+For example, for a trait involving the 'age at which disease manifestations first appear', fill in the table [`disease_onset.tsv`.](https://github.com/obophenotype/bio-attribute-ontology/blob/master/src/patterns/data/default/disease_onset.tsv).
+Create a unique OBA identifier by using the next available ID from your [assigned range.](https://github.com/obophenotype/bio-attribute-ontology/blob/master/src/ontology/oba-idranges.owl)
+
+Keep track of the IDs from your range that you have already assigned.
+
+Also fill in the appropriate terms as specified in the actual DOS-DP `yaml` template file.
+For example, in the case of the [`disease_onset.tsv`.](https://github.com/obophenotype/bio-attribute-ontology/blob/master/src/patterns/data/default/disease_onset.tsv) table, you must use MONDO [disease or disorder](https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0000001) terms in the `disease` column.
```
Comment (matentzn) [inline docs/SOP-add_OBA_terms_for_GWAS.md:19]: You are missing instructions on making a pull request with the changes and merging them into master

File: docs/SOP-add_OBA_terms_for_GWAS.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+* In a terminal window, start the [release pipeline:](http://pato-ontology.github.io/pato/odk-workflows/ReleaseWorkflow/)
+
+```shell
+sh run.sh make prepare_release -B
+```
+
+* Alternatively, if you do not need to refresh all the imports, run a fast release:
+
+```shell
+sh run.sh make prepare_release_fast
```
Comment (matentzn) [inline docs/SOP-add_OBA_terms_for_GWAS.md:48]: This should be the recommended method. We should decouple releases from imports

File: docs/SOP-add_OBA_terms_for_GWAS.md (hunk 1 above)
Comment (rays22) [reply docs/SOP-add_OBA_terms_for_GWAS.md:39]: Added a new document `OBA_ReleaseWorkflow.md`.

File: docs/SOP-add_OBA_terms_for_GWAS.md (hunk 2 above)
Comment (rays22) [reply docs/SOP-add_OBA_terms_for_GWAS.md]: Added some instructions on 'PR-->master' github workflow.

**Pull request merged**